### PR TITLE
Point to the feedstock.

### PR DESCRIPTION
**Pull request by ocefpaf**
Should fix the failures in our CIs.

### CODE REVIEW

**Comment by pelson** [comment]
Hmmm. Not sure what's going on here.

**Comment by ocefpaf** [comment]
I cannot see the problem either.  (And if I cannot find it I'll just remove netcdf4 from IOOS and start educating our people on conda-forge :smile:)

**Comment by ocefpaf** [comment]
Found the problem.  The mystery is how `AppVeyor` passed!

**Comment by pelson** [comment]
What was it? I'm curious.

File: netcdf4/source.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-gh-repo: conda-forge/staged-recipes
+gh-repo: conda-forge/netcdf4-feedstock
 git-tag: master
-contents: recipes/netcdf4/*
+contents: recipe/*
```
Comment (pelson) [inline netcdf4/source.txt]: Ah, was this still netcdf4?

Comment (ocefpaf) [inline netcdf4/source.txt]: Yep.  That was my mistake.

**Pull request merged**